.github/workflows/prepare_release.yml

```diff
@@ -46,6 +46,19 @@ jobs:
               .toString("utf-8")
               .split("\n");
             const newContent = changes.concat(rawContent).join("\n");
+
+            const { data: master } = await github.rest.git.getRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: "heads/master",
```

Comment:
should we use the base from which the `workflow_dispatch` was triggered?

Reply:
No I don't think we needed that until now.